### ISSUE TITLE
Circuit unrolling batches

### ISSUE THREAD

**Issue by Jacfomg** [qiboteam/qibolab#706]
Can we make circuit unrolling use the some kind of batches to fit in memory like we did for the sequences @stavros11 ?

**Comment by Jacfomg** [comment]
Yes, then it is my bad, the memory limit could be quite wrong when executing things with some several drive pulses.

**Comment by stavros11** [comment]
Yes, this is also something we need to check. For qblox I didn’t notice much difference from drive pulses but I didn’t try very very many.

**Comment by Jacfomg** [comment]
I was running some RB with unrolling and with decent size depth for the circuits it reduced by a factor of 5 to 10 compared to simpler calibration sequences like ramsey.
